### DISCUSSION TITLE
Thank you guys!

### DISCUSSION THREAD

**Discussion by julielangou** [jekyll/jekyll#8834] [Q&A]
It is very very touching to dedicate your 4.2.1 Release to Frank
Please keep going on your Open Source project.. 
Frank will be so proud.

Frank's sister, Julie
working on Open Source projects too ;-)
julie.langou - @ - gmail.com

**Comment by ashmaroli** [comment]
Thank you for reaching out to us, Julie!!
Frank was a big part of Jekyll and the community around it. Words will not be enough to express our gratitude.
Dedicating our latest release is the least we can do to honor his legacy.

Hope the family has found the strength to move on with Life.
Best Wishes for your Open Source projects, Julie.

Ashwin M
